models/DataObject/ClassDefinition/Data/User.php

```diff
@@ -179,4 +206,51 @@ public function setUnique($unique)
     {
         $this->unique = $unique;
     }
+
+    /**
+     * @param string|null $restrictTo
+     *
+     * @return $this
+     */
+    public function setRestrictTo($restrictTo)
+    {
+        /**
```

Comment:
where is this comment copied from? this can be removed :-)

Reply:
Okay 👍 I have oriented me on the selection of countries